### ISSUE TITLE
Finer control when displaying a directory

### ISSUE THREAD

**Issue by anntzer** [derf/feh#513]
`feh -R<somenumber> $directory` is a very useful feature :-)
It could be made even more useful if it was possible to instead pass an unexpanded glob pattern as argument (likely with an additional flag to disambiguate), as sometimes directories contain not only images, but also other files.
Alternatively, `feh` could provide a flag like `--skip-unloadable` that at least silences imlib2's warning for non-loadable files (which may be present in the directory but that the user simply doesn't want to display).

Thanks for the nice program!

**Comment by c99pedant** [comment]
To quickly get rid of imlib2 warnings you could redirect `stderr` to `/dev/null` which would look like: `feh -R 30 /opt/pub/img 2>/dev/null;` You'd just need to be careful if there are any other warnings or errors which would also be hidden.

**Comment by anntzer** [comment]
Fair enough, feel free to close if you're not going to implement the requested feature then (I agree 2>/dev/null may be enough).
